src/CONFIG.js

```diff
@@ -49,11 +49,17 @@ export default {
     EXPENSIFY: {
         // Note: This will be EXACTLY what is set for EXPENSIFY_URL whether the proxy is enabled or not.
         EXPENSIFY_URL: expensifyURL,
-        SECURE_EXPENSIFY_URL: secureURLRoot,
         NEW_EXPENSIFY_URL: newExpensifyURL,
-        URL_API_ROOT: expensifyURLRoot,
-        STAGING_EXPENSIFY_URL: stagingExpensifyURL,
-        STAGING_SECURE_EXPENSIFY_URL: stagingSecureExpensifyUrl,
+
+        // The DEFAULT API is the API used by most environments, except staging, where we use STAGING (below)
+        // The "staging toggle" in settings toggles between DEFAULT and STAGING APIs
+        // We can think of DEFAULT as PROD, because on both STAGING and PROD it points to the production address
```

Comment:
```suggestion
On both STAGING and PROD it points to the production address
```

I think this is enough. Saying "we can think of DEFAULT as PROD" is unclear advice and we immediately contradict it on the next line by saying it could be other things. It's not necessarily pointing to the production API. It can be several things.